userguide/payment/includes/payment-states.adoc

```diff
@@ -55,29 +55,30 @@ Note that the first 3 cases are normal cases but the last 2 are errors that are
 
-The Janitor refers to a set of internal Kill Bill features to help ensure transactions end up in a terminal state: these are the Janitor notification queue and the on-the-fly Janitor. They both work by calling the plugin (via `getPaymentInfo` PaymentPluginApi) and updating the state owned by Kill Bill if it has changed. It is imperative payment plugin developers take great care when implementing this API for this mechanism to work (see our Payment Plugin developer documentation for more details).
+The Janitor refers to a set of internal Kill Bill features to help ensure transactions end up in a terminal state: these are the Janitor notification queue and the on-the-fly Janitor. They both work by calling the plugin (via https://github.com/killbill/killbill-plugin-api/blob/d9eca5af0e37541069b1c608f95e100dbe13b301/payment/src/main/java/org/killbill/billing/payment/plugin/api/PaymentPluginApi.java#L144[`PaymentPluginApi.getPaymentInfo`]) and updating the state owned by Kill Bill if it has changed. It is imperative payment plugin developers take great care when implementing this API for this mechanism to work (see our Payment Plugin developer documentation for more details).
 
 ==== Janitor notification queue
 
 A Janitor notification entry is inserted whenever a payment results in a non terminal state (whether the payment was initiated internally by Kill Bill or externally via our payment APIs). Kill Bill will then periodically poll the plugin to attempt to fix the state.
 
-The polling delay is configured by `org.killbill.payment.janitor.unknown.retries` for UNKNOWN transactions (default `5m,1h,1d,1d,1d,1d,1d`) and `org.killbill.payment.janitor.pending.retries` for PENDING transactions (default `1h, 1d`). Companies relying on PENDING for offline payment methods (ACH, Boleto, etc.) should increase the number of attempts, as such payments can be slow (they can take multiple days to execute).
+The polling delay is configured via the  `org.killbill.payment.janitor.unknown.retries` property for UNKNOWN transactions (default `5m,1h,1d,1d,1d,1d,1d`) and the `org.killbill.payment.janitor.pending.retries` property for PENDING transactions (default `1h, 1d`). These properties can be configured as explained in the https://docs.killbill.io/latest/userguide_configuration.html[_Configuration Guide_]. Companies relying on PENDING for offline payment methods (ACH, Boleto, etc.) should increase the number of attempts, as such payments can be slow (they can take multiple days to execute).
 
 ==== On-the-fly Janitor
```

Comment:
```suggestion
==== On-The-Fly Janitor
```
I checked a capitalization tool, and most style guides capitalized the "the" (because it's part of the overall descriptive adjective, I suppose).